### ISSUE TITLE
TCC：Stop early when I was training

### ISSUE THREAD

**Issue by hubin111** [google-research/google-research#97]
I1031 20:37:34.190690 140505617327872 train.py:123] Iter[27/150000], 27.4s/iter, Loss: 0.708
I1031 20:38:02.018666 140505617327872 train.py:123] Iter[28/150000], 27.8s/iter, Loss: 1.107
I1031 20:38:29.308702 140505617327872 train.py:123] Iter[29/150000], 27.3s/iter, Loss: 0.724
I1031 20:38:56.431218 140505617327872 train.py:123] Iter[30/150000], 27.1s/iter, Loss: 0.744

Process finished with exit code -1
Why did I stop early when I was training? I didn't terminate the program, it ended automatically, and I couldn't continue training. excuse me, can you tell me the reason

**Comment by debidatta** [comment]
Very strange. Time per iteration is also quite high. Are you using a GPU to run the training script?

**Comment by hubin111** [comment]
Fri Nov  1 14:39:50 2019       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 430.40       Driver Version: 430.40       CUDA Version: 10.1     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  TITAN Xp            Off  | 00000000:02:00.0 Off |                  N/A |
| 23%   35C    P8     9W / 250W |    157MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   1  TITAN Xp            Off  | 00000000:03:00.0 Off |                  N/A |
| 25%   41C    P8    10W / 250W |    157MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   2  TITAN Xp            Off  | 00000000:82:00.0 Off |                  N/A |
| 34%   60C    P2   273W / 250W |  11774MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   3  TITAN Xp            Off  | 00000000:83:00.0 Off |                  N/A |
| 40%   67C    P2   238W / 250W |   6914MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0     32902      C   /root/userfolder/anaconda3/bin/python        147MiB |
|    1     32902      C   /root/userfolder/anaconda3/bin/python        147MiB |
|    2     32902      C   /root/userfolder/anaconda3/bin/python        147MiB

**Comment by debidatta** [comment]
Thanks for sharing this. Are you setting the GPUs visible to this script or changing the number of GPUs while setting distribution strategy? By default the script should be using all GPUs on machine. Even then it shouldn't take so long for one iteration. Can you print the batch size being fetched by the dataset (total_batch_size in train.py)? Also can you share the full logs?

**Comment by hubin111** [comment]
Thank you very much for your help, I have found the problem caused by the server.
